slick2d-hiero/pom.xml

```diff
@@ -64,12 +69,15 @@
 			</plugin>
 			<plugin>
 				<artifactId>maven-jar-plugin</artifactId>
+				<version>3.0.0</version>
 				<configuration>
 					<archive>
 						<manifest>
 							<mainClass>org.newdawn.slick.tools.hiero.Hiero</mainClass>
 							<addClasspath>true</addClasspath>
 							<classpathPrefix>dependency/</classpathPrefix>
+							<classpathLayoutType>custom</classpathLayoutType>
+							<customClasspathLayout>${artifact.artifactId}-${artifact.version}${dashClassifier?}.${artifact.extension}</customClasspathLayout>
```

Comment:
I'm not sure why that's necessary since it just points to the JAR, which should be the default?